integration-tests/src/test/resources/model-in-image/create-domain.sh

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env bash
```

Comment:
@mriccell @vanajamukkara @tbarnes-us @ddsharpe  This is a big script to be part of tests.  It looks like product code.

Reply:
Looking inside this script, it looks like a repeat of the MII sample scripts which could be used instead to do the same tasks.  I think this is a side effect of the QA tests and sample scripts being developed at the same time.  But I don't mind that this script doesn't depend on the MII sample scripts - maybe that's a good thing.  Note that there are dedicated tests for directly testing the MII sample scripts.

Reply:
@rjeberhard  Please resolve if you think this comment is being addressed now.